scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1247,6 +1247,25 @@ public String getTaskServerLogsByTag(String jobId, String taskTag)
                                  sortParameters);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @ImmediateService
+    public List<JobInfo> getJobsInfoList(List<String> jobsId)
+            throws PermissionException, NotConnectedException, UnknownJobException {
+        frontendState.checkPermission("getJobsInfoList", "You don't have permissions to load jobs");
```

Comment:
Actually, this part is more complicated.
you should check the permission for each job inside the loop.

Via:
checkPermissions("getJobState",
                         getIdentifiedJob(jobId),
                         YOU_DO_NOT_HAVE_PERMISSION_TO_GET_THE_STATE_OF_THIS_JOB);